imports/ui/signup/signup.jsx

```diff
@@ -9,14 +10,43 @@ import PasswordInput from '../components/password-input'
 import actions from './signup.actions'
 import LoginLayout from '../layouts/login-layout'
 import { emailValidator } from '../../util/validators'
+import CircularProgress from 'material-ui/CircularProgress'
+
+type Props = {
+  dispatch: (info: {}) => void,
```

Comment:
`dispatch` receives actions. I think it would be more concise to name the arguments `action` and not `info`